cmd/root.go

```diff
@@ -183,6 +183,8 @@ func AddCommands(cmd *cobra.Command) {
 		config.Beacon.WeakCoinRoundDuration, "Weak coin round duration in milliseconds")
 	cmd.PersistentFlags().Var((*types.RatVar)(config.Beacon.Theta), "beacon-theta",
 		"Ratio of votes for reaching consensus")
+	cmd.PersistentFlags().IntVar(&config.Beacon.ProposalNumBytes, "beacon-proposal-num-bytes",
```

Comment:
in which case we want to change this? can we define size as a constant?

Reply:
this is per request by Iddo in slack https://spacemesh.slack.com/archives/CG91F13C7/p1642773410106600?thread_ts=1642514200.091200&cid=CG91F13C7

Reply:
but maybe it's better to name it `beacon-num-bytes` instead

Reply:
i don't think that he meant it to be a parameter for a node binary, that wouldn't make any sense

Reply:
i think if we don't intend to change something after binary is compiled it shouldn't be configurable. what matters from research pow is that this can be changed in general, not for each network. we already have a bunch of useless parameters

Reply:
music to my ears. happy to oblige. changed it to using a constant. thanks